todo_app/mongo.py

```diff
@@ -0,0 +1,87 @@
+import os
+from bson.objectid import ObjectId
+import pymongo 
+from dotenv import load_dotenv
+from todo_app.todo_card import TodoCard
+from datetime import datetime
+
+class Mongo:
+    def __init__(self): 
+        self.user_name=os.getenv('USER_NAME')
+        self.password=os.getenv('PASSWORD')
+        self.mongo_url=os.getenv('MONGO_URL')
+        self.default_database=os.getenv('DEFAULT_DATABASE')
+        self.board_name=os.getenv('BOARD_NAME')
+        self.connection_string = "mongodb+srv://" + self.user_name + ":" + self.password + "@" + self.mongo_url + "/" + self.default_database + "?w=majority"
+
+    def getClient(self):
+        return pymongo.MongoClient(self.connection_string)
```

Comment:
Apart from inside the `get_databases` function, every time you call getClient, what you're _actually_ interested in is the database.

You have this block of code multiple times:
```
        client = self.getClient()
        to_do_app_database = client["To_Do_App"]
        to_do_items_collection = to_do_app_database['ToDoItems']
```

It would make more sense to have a function `get_database(self)`

Reply:
done, added method to return collection